Server/src/main/webapp/index.html

```diff
@@ -4,35 +4,55 @@
     <title>Opinion</title>
     <meta charset="utf-8">
     <link rel="stylesheet" type="text/css" href="resources/css/styles.css">
-    <script src="resources/js/script.js"></script>
+    <script src="resources/js/model.js"></script>
+    <script src="resources/js/view.js"></script>
+    <script src="resources/js/controller.js"></script>
+    <script src="resources/js/fillLocalStorage.js"></script>
+
 </head>
 <body>
 <header id="header">
     <img class="logo" src="Logo.png">
-    <input class="input-field" placeholder="Find your topic" oninput="SelectController.onTagsInput(this.value)">
-    <div class="tags-select" id="tagSelect">
-        <template id="option">
-        </template>
-    </div>
-    <p class="my-profile">Andrei</p>
-    <button class="button log-out"><p class="button-text">Log out</p></button>
-    <button class="button menu" onclick="HeaderView.displayFiltration()"><img src="menu.png" class="menu icon"></button>
-    <form id="filtration">
-        <div class="filtration">
-            <label class="user-name-filter" for="userName">User name<input id="userName" type="text" placeholder="name" class="user-name-filter input"></label>
-            <label class="date-from-filter" for="dateFrom">Date from:<input id="dateFrom" type="text" placeholder="02 03 2020" class="date input"></label>
-            <label class="date-to-filter" for="dateUntil">Date until:<input id="dateUntil" type="text" placeholder="02 03 2020" class="date input"></label>
+    <p class="my-profile"></p>
+    <button id="login-button" class="button log-out"><p class="button-text">Log out</p></button>
```

Comment:
For button you don't need inside tag `p` with class, you can specify these styles for the button